Userland/Utilities/CMakeLists.txt

```diff
@@ -14,7 +14,8 @@ list(APPEND RECOMMENDED_TARGETS
 # FIXME: Support specifying component dependencies for utilities (e.g. WebSocket for telws)
 
 foreach(CMD_SRC ${CMD_SOURCES})
-    get_filename_component(CMD_NAME ${CMD_SRC} NAME_WE)
```

Comment:
note for the future, NAME_WLE is supported since CMake 3.14

```
LAST_EXT  = File name last extension (.c from d/a.b.c)
NAME_WLE  = File name with neither the directory nor the last extension
```